lib/utils/logger/index.spec.js

```diff
@@ -0,0 +1,42 @@
+const {getLogger} = require('./index');
+
+describe('utils/logger', () => {
+    let mockConsole;
+    let logger;
+
+    beforeAll(() => {
+        mockConsole = {
+            content: '',
+
+            log: function(...args) {
+                this.content = [...args];
+            },
+        }
+        logger = getLogger(mockConsole);
+    });
+
+    test('Testing normal log message', () => {
+
+        logger.log('test', 'plop');
```

Comment:
Why not use mock functions from Jest?
https://jestjs.io/docs/en/mock-functions 

Then you could `getLogger` with `{ log: jest.fn() }`
```javascript
const mockConsole = {
  log: jest.fn(),
};

logger = getLogger(mockConsole);

logger.log('Pizza')

expect(mockConsole.log.mock.results[0].value).toBe('Pizza')
```